testing/internal-integ-testing/src/main/groovy/org/gradle/test/fixtures/server/http/BlockingHttpServer.java

```diff
@@ -471,6 +488,13 @@ public interface BuildableExpectedRequest extends ExpectedRequest {
          */
         BuildableExpectedRequest broken();
 
+        /**
+         * Sends a 500 response with some arbitrary content as the response body.
+         *
+         * @return this
```

Comment:
Hi! I found out while looking at your PR, you might have missed updating this!

Reply:
Hi @jknair0,
thank you, I fixed this comment :smiley: